### ISSUE TITLE
Sphinx 1.1.2 not working on python2.4

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#822]
Sphinx-1.1 series will deprecate Python-2.4 support?

on Python-2.4, sphinx-quickstart raise syntax error:

```
#!python

(sphinx1124) D:\sphinx1124>sphinx-quickstart

  File "D:\sphinx1124\lib\site-packages\sphinx-1.1.2-py2.4.egg\sphinx\util\osutil.py", line 145
    return docname.split(SEP, 1)[0] if compaction else docname
                                     ^
SyntaxError: invalid syntax
```

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/822
- Originally reported by: Takayuki Shimizukawa
- Originally created at: 2011-11-15T05:40:59.086

**Comment by shimizukawa** [comment]
_From [jcmcken](https://bitbucket.org/jcmcken) on 2011-12-18 01:53:31+00:00_

Patch attached.

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2012-01-29 10:37:17+00:00_

Fixed in <<cset 1436989b6bccd1c2224465b33cd2a75ca59f95b8>>.
